otsclient/git_gpg_wrapper.py

```diff
@@ -30,7 +31,7 @@
 def main():
     parser = otsclient.args.make_common_options_arg_parser()
 
-    parser.add_argument("-g", "--gpg-program", action="store", default="/usr/bin/gpg",
+    parser.add_argument("-g", "--gpg-program", action="store", default=shutil.which("gpg") or "/usr/bin/gpg",
```

Comment:
```suggestion
    parser.add_argument("-g", "--gpg-program", action="store", default="gpg",
```

Looks like `Popen` already searches in `$PATH` for us, so if we're going to use `$PATH` we don't need to figure it out ourselves.

Reply:
Okay. But then it was also unnecessary to provide `--gpg-program "`which gpg`"` in `ots-git-gpg-wrapper.sh`.

Reply:
A nice side-effect of specifying the `shutil.which()`-found path is that the help page will show exactly which gpg program will be used. This wouldn't be the case if we just let `Popen` find it.